app/models/foreman_openscap/arf_report.rb

```diff
@@ -13,8 +13,13 @@ class ArfReport < ::Report
     has_one :policy, :through => :policy_arf_report
     has_one :asset, :through => :host, :class_name => 'ForemanOpenscap::Asset'
     after_save :assign_locations_organizations
+    has_one :log, :foreign_key => :report_id
+
+    delegate :result, :to => :log, :allow_nil => true
```

Comment:
why?
there is no result method on ArfReport. as I said, I don't understand how this worked on rails 3, if anyone can explain this to me (and offer a working solution), I'd be more than happy to remove this code :smile:

without this line, ArfReports can't be created, as the validation tries to call #result, only to receive a NoMethodError